map/transit/transit_reader.hpp

```diff
@@ -98,11 +50,12 @@ class ReadTransitTask: public threads::IRoutine
 public:
   ReadTransitTask(Index & index,
                   TReadFeaturesFn const & readFeaturesFn)
-    : m_transitReader(index), m_readFeaturesFn(readFeaturesFn)
+    : m_index(index), m_readFeaturesFn(readFeaturesFn)
   {}
 
-  bool Init(uint64_t id, MwmSet::MwmId const & mwmId, std::unique_ptr<TransitDisplayInfo> && transitInfo = nullptr);
+  void Init(uint64_t id, MwmSet::MwmId const & mwmId, std::unique_ptr<TransitDisplayInfo> && transitInfo = nullptr);
```

Comment:
cstdint?

Reply:
What's the reason to pass unique_ptr by rvalue? Pass it by value.

Reply:
> cstdint?

Done

> What's the reason to pass unique_ptr by rvalue? Pass it by value.

I would pass unique_ptr by value. @darina , do you mind?

Reply:
I don't mind.

Reply:
> What's the reason to pass unique_ptr by rvalue? Pass it by value.

Done